tests/conftest.py

```diff
@@ -30,6 +30,17 @@ def serial_driver(target, serial_port, monkeypatch):
     s = SerialDriver(target)
     return s
 
+@pytest.fixture(scope='function')
+def onewire_port(target):
+    return OneWirePIO(target, "testhost", "path")
+
+@pytest.fixture(scope='function')
+def onewire_driver(target, onewire_port, monkeypatch):
+    onewire_mock = MagicMock
+    import onewire
+    monkeypatch.setattr(onewire, 'Onewire', onewire_mock)
+    s = OneWirePIODriver(target)
+    return s
 
```

Comment:
Either move these fixtures to test_onewire.py or configure the mock in the test itself. Also use pytest-mock instead of unittest.mock.